Documentation/ceph-cluster-crd.md

```diff
@@ -197,13 +195,13 @@ This allows to keep Rook components running when for example a node runs out of
 You can set resource requests/limits for Rook components through the [Resource Requirements/Limits](#resource-requirementslimits) structure in the following keys:
 
 - `mgr`: Set resource requests/limits for MGRs.
-- `mon`: Set resource requests/limits for Mons.
+- `mon`: Set resource requests/limits for MONs.
```

Comment:
The trend I have been following is to only fully capitalize acronyms. Monitors are mons , managers are mgrs, MetaData Servers are MDSes, RADOS GatWeays are RGWs, Object Storage Daemons are OSDs, RADOS Block Devices are RBDs etc.

Reply:
Alright! I'll follow your style. Maybe it would make sense to create a `doc-style-guide.md` or something similar, with the preferred capitalization, spelling etc. of common words?